pages/common/git-browse-ci.md

```diff
@@ -0,0 +1,13 @@
+# git browse-ci
+
+> Opens the current `git` repository's CI on websites such as Git.
```

Comment:
Can we change this description a bit?
```suggestion
> Opens the current `git` repository's CI website in your default web browser.
```
Or 

```suggestion
> Opens the current `git` repository's CI on websites such as Git in your default web browser.
```
What do you guys think?

Reply:
_it should be GitHub anyway instead of Git_

I think the second option is best.